electron/main/index.ts

```diff
@@ -253,7 +253,17 @@ async function createWindow() {
             | Ada
             | Tada
             | Dot
-            | Tdot;
+            | Tdot
+          return await mpcCoin.recoverConsolidations(params);
+        }
+        case 'sol':
+        case 'tsol': {
+          if (!('durableNonces' in params)) {
```

Comment:
I think this object always exist, since you are definning it as 
```
 durableNonces: {
        publicKeys: '',
        secretKey: '',
      }
```
Maybe should check that the prv and pubs are not empty

Reply:
this is really just to make the typing engine happy but I can add that in